### PR TITLE
Add margin variable for breadcrumb

### PR DESCRIPTION
**Pull request by prateekgoel**
I have added a variable for breadcrumb margin to `_variables.scss` and replaced static margin in `_reboot.css` with that variable.

Fixes https://github.com/twbs/bootstrap/issues/23016.

### CODE REVIEW

File: scss/_breadcrumb.scss [hunk 1]
```diff
@@ -1,6 +1,6 @@
 .breadcrumb {
   padding: $breadcrumb-padding-y $breadcrumb-padding-x;
-  margin-bottom: 1rem;
```
Comment (mdo) [inline scss/_breadcrumb.scss]: Leave it as `margin-bottom`—there's no need to reset a non-existant `margin-top`, `-right`, and `-left` if we have no intention of ever applying those values (which I don't :p).

Comment (prateekgoel) [inline scss/_breadcrumb.scss]: Ok then i'll keep `margin-bottom` only.
Will do the change.

Comment (prateekgoel) [inline scss/_breadcrumb.scss]: @mdo Changes done

Comment (prateekgoel) [inline scss/_breadcrumb.scss]: @mdo Can you please review and close this PR?

**Comment by mdo** [comment]
This'll go in for Beta 2 at the earliest.

**Comment by prateekgoel** [comment]
Ok Thanks @mdo

**Pull request merged**